API/api/helperClasses/enums/validation.php

```diff
@@ -5,12 +5,18 @@
      * PHP file containing all defined regex validations.
      * 
      * author: Maximilian T. | Kontr0x
-     * last edit / by: 2021-05-26 / Maximilian T. | Kontr0x
+     * last edit / by: 2021-06-01 / Maximilian T. | Kontr0x
      */
 
     abstract class Validation{
         const ApiKey = '/^[0-9a-zA-Z]{20}$/';
         const Session = '/^[0-9a-zA-Z]{20}$/';
         const UserName = '/^[0-9a-zA-Z]{3,20}$/';
+        const BooleanValue = '/^(?:True){1}$|^(?:False){1}$/';
+        const Overtime = '/^(?:[0-9]|[1-9][0-9]|[1-9][0-9][0-9])$/';
+        const WeeklyWorkingMinutes = '/^(?:[0-9]|[1-9][0-9]|[1-9][0-9][0-9]|[1-2][0-9][0-9][0-9]|[1-3][0]{3})$/';
+        const WeeklyWorkingDays = '/^(?:[0-6]{1})$/';
```

Comment:
I think weekly working days should go up to 7 days? There are some rare situations with Sunday working contracts. What do you think @TrickyTreat @an055?

Reply:
Yes, I think 7 days would be good. Then we can cover every possibility.